docs/userGuide/contentAuthoring.md

```diff
@@ -87,27 +87,27 @@ In addition, it supports:
 
 * Radio Button List
 
-  ```
+  ```html
   - ( ) Option 1
   - (X) Option 2 (selected)
   ```
 
 * Youtube video
-  ```
+  ```html

```

Comment:
Since the following content is all in markdown, marking it with `markdown` instead of `html` will allow it to highlight properly (right now this does not have any highlight at all).

Reply:
Thanks for catching those!